elements/rh-tooltip/rh-tooltip.css

```diff
@@ -0,0 +1,14 @@
+:host([on=dark]) {
+  --pf-c-tooltip__content--BackgroundColor: #ffffff;
+  --pf-c-tooltip__content--Color: #151515;
+}
+
+:host {
+  --pf-c-tooltip--BoxShadow: 0 0.125rem 0.25rem 0 rgba(21, 21, 21, 0.2);
```

Comment:
reserving this comment pending @marionnegp's design review.

Resolve this (and any subsequent) comment threads to auto-merge